resources/lang/de_DE/default.po

```diff
@@ -2089,8 +2092,8 @@ msgid ""
 "Here is the list of teams and their tasks. If you have questions, read the "
 "FAQ."
 msgstr ""
-"Hier ist die Liste der Teams und ihrer Aufgaben. Wenn Du Fragen hast, schaue "
-"im FAQ nach."
+"Hier ist die Liste der Engeltypen und ihrer Rechte. Wenn Du Fragen hast, "
```

Comment:
engel?

Reply:
I didn't change the terms `angel`/`Engel` and `angeltype`/`Engeltyp`. We cloud of course change those terms to something else.